docs/markdown/BASIC_INSTALL.md

```diff
@@ -1,51 +1,82 @@
-# GridPACK step=by-step installation instructions
```

Comment:
The steps here need to be updated to be consistent with the installation in `install_gridpack_deps.sh` and `install_gridpack.sh`